packages/frontend/scss/composer/_composer.scss

```diff
@@ -79,7 +79,7 @@
 
     .emoji-button {
       height: 40px;
-      padding: 0px 3px;
+      padding: 0px 8px 0px 3px;
```

Comment:
```suggestion
      padding: 0px 3px;
      margin-inline-end: 5px;
```

This makes it look like this

![image](https://github.com/user-attachments/assets/d906691e-5464-4059-8ab6-32882d709f24)


Otherwise it's like this 

![image](https://github.com/user-attachments/assets/eac9d142-f20e-4808-96c5-66d1e87684b8)